### DISCUSSION TITLE
How to build a detector without a box but single points

### DISCUSSION THREAD

**Discussion by Djennifer** [Project-MONAI/MONAILabel#1460] [Q&A]
Hi all,

I have a question regarding the different inference types. I have a module that predicts multiple landmarks on an image and I am struggling to bring this in an interactive setting with MONAILabel and 3D Slicer. Every point from my model has an x, y and z coordinate and I think it would be nice if they could be visualized with the markup tool. However, I am unsure about which interface in MONAILabel to use for this. I noticed that the detection inference type is using boxes, so does somebody know how I could visualize single points? I think I need to create my own custom writer. Where would I need to pass that?

**Comment by SachidanandAlle** [comment]
Deepgrow
